tuxemon/core/states/world/worldstate.py

```diff
@@ -825,7 +824,7 @@ def check_interactable_space(self):
                         tile = (player_tile_pos[0] - 1, player_tile_pos[1])
                     elif direction == "right":
                         tile = (player_tile_pos[0] + 1, player_tile_pos[1])
-                    for npc in self.npcs:
+                    for slug,npc in self.npcs:
```

Comment:
This line should be:

```python
for slug, npc in self.npcs.items()
    ...
```

Otherwise you get this error:
```python
>>> npcs = {}
>>> npcs["npc_oak"] = "Oak!"
>>> for slug, npc in npcs:
...     print(slug, npc)
... 
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: too many values to unpack
```